yellow-paper/docs/l1-smart-contracts/index.md

```diff
@@ -33,7 +33,7 @@ def process(block: ProvenBlock, proof: Proof):
     assert self.outbox.insert(
         block_number, 
         header.content_commitment.out_hash, 
-        header.content_commitment.tx_tree_height
+        header.content_commitment.tx_tree_height + 1
```

Comment:
Why the "+1" here?

Reply:
It is the height of the outgoing tree, which is the heigh of the transactions tree + 1, because each tx have 2 messages. Should probably be something more like `math.ceil(log2(MAX_NEW_L2_TO_L1_MSGS_PER_TX))`